src/tutorials/terrain_reference.md

```diff
@@ -0,0 +1,510 @@
+
+![Png Example](../img/TerrainReference/PngExample.png){: .center loading="lazy" }
+
+Weightmaps use these four color channels to automatically paint the terrain. Weightmaps tie each of the four color channels of your `.png` file to the four materials associated with your terrain.
+
+- The **red**{: .Color_RED } channel is tied to the *first* material
+- The **green**{: .Color_GREEN } channel is tied to the *second* material
+- The **blue**{: .Color_BLUE } channel is tied to the *third* material
+- The **alpha** channel is tied to the *fourth* material
+
+<div style="display: flex; flex-direction:row; justify-content: center">
+    <div style="display:block; flex:2.144">
+        <img alt="Weightmap Demo"src="../../img/TerrainReference/SplatMapDemo.png" style="max-height:100%; height:100%; max-width:100%; width:100%;"/>
+    </div>
+    <div style="display:block; flex:1;">
+        <img alt="Weightmap" src="../../img/TerrainReference/Splatmap.png" style="max-height:100%; height:100%; max-width:100%; width:100%;"/>
+    </div>
+</div>
+<p style="text-align:center">
+    <em>
+        <em>Materials Applied to the terrain</em> (left) vs. <em>Source Weightmap (right)</em>
+    </em>
+</p>
+
+!!!note
+    If your weightmap `.png` file uses the **alpha** channel, then you must enable the **Import Alpha Channel** property in the **IMPORT TERRAIN HEIGHTMAP** window before generating your terrain.
+
+    ![Import Alpha Channel](../img/TerrainReference/ImportAlphaChannelWindow.png){: .center loading="lazy" style="max-height:14em" }
+
+To import your weightmap, you must follow the same steps you followed for importing the heightmap. In the **IMPORT TERRAIN HEIGHTMAP** window click on the ![load file icon](../img/TerrainReference/ImportIcon.png)icon next to the **Weightmap** field and select an `.png` file for your weightmap. Then click on the **Import** button.
+{: .image-inline-text .image-background }
+
+![Import Weightmap File](../img/TerrainReference/ImportWeightmapIcon.png){: .center loading="lazy" }
+
+![Click on Import Button](../img/TerrainReference/ImportPNGMap.png){: .center loading="lazy" }
+
+Weightmap images need to square `.png` images. The **Terrain Generator** will only accept weightmap images of the following sizes:
+
+- 128 x 128
+- 256 x 256
+- 512 x 512
+- 1024 x 1024
```

Comment:
Need a Learn More section. You can copy it from another section, but repeat any links you used earlier to things on docs. The Environment Art tutorial would be one, Materials ref. Anything that you might have gotten here but it didn't